openeo/rest/connection.py

```diff
@@ -1355,3 +1356,37 @@ def paginate(con: Connection, url: str, params: dict = None, callback: Callable
         page += 1
         params = {}
 
+
+
+#
+# For caching connections
+#
+
+_default_url = "openeo-dev.vito.be"
```

Comment:
Avoid adding these things here, they're not strictly necessary for the job manager, and too specific for certain backends.

Reply:
Code for connection caching was no longer relevant, so I removed it.